### ISSUE TITLE
When using Command + up/down keys, arrows on-screen stick and stay at either 1 or 10

### ISSUE THREAD

**Issue by brooklynlash** [phetsims/gravity-force-lab-basics#309]
**Test device**
Macbook Pro

**Operating System**
Mac 10.15.7

**Browser**
Chrome

**Problem description**
For https://github.com/phetsims/qa/issues/702
I tried using Command + up/down to increase or decrease the size of the mass balls on the mass sliders while on keyboard navigation, but once you click a combo once, the arrow sticks at that position and will not allow you to change the size of mass at all unless you tab over to another button and come back or reset the sim entirely. Sometimes it resets after spamming your keys a bit but of course that is not ideal. 

**Steps to reproduce**
1. Use the tab key to navigate to either mass 1 or 2
2. Click either command+up keys or command+down keys simultaneously

**Visuals**
An attempt at displaying this while Katie videos me...
https://user-images.githubusercontent.com/72031637/132584703-f76a0688-9648-494d-9c9d-17a9de2d9db0.mp4

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Language: en-US
Window: 1680x868
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @brooklynlash. I was curious if this was introduced in a recent change of how we determine which keys are pressed, but I just noticed that this issue is in the published version as well so it has likely been an issue for longer than that.

**Comment by jessegreenberg** [comment]
It appears that we do not get the keyup event when the meta key is down:
https://stackoverflow.com/questions/11818637/why-does-javascript-drop-keyup-events-when-the-metakey-is-pressed-on-mac-browser

**Comment by jessegreenberg** [comment]
The same is sort of true for Windows, but the meta key for windows removes focus from the document so this isn't possible. I think the best thing we can do is prevent any keydown behavior when a meta key is down so that we cannot get in this state.

**Comment by jessegreenberg** [comment]
We have encountered this before in KeyboardDragListener and handled it with 
https://github.com/phetsims/scenery/blob/d73d6fc643306e149c3eede8706e6d107c978397/js/listeners/KeyboardDragListener.js#L265-L271

**Comment by jessegreenberg** [comment]
A fix for this has been made in the above commit for both AccessibleValueHandler and AccessibleNumberSpinner. I tested myself on Windows Chrome and Firefox and MacOS Chrome and Firefox and the behavior is fixed.

**Comment by jessegreenberg** [comment]
Cherry pick done.

**Comment by Nancy-Salpepi** [comment]
On Mac + chrome, pressing command + arrow keys no longer changes the value in the vertical spinner.
